src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -475,13 +475,21 @@ class ResourceService(
     for {
       userEmails <- directoryDAO.loadUsers(users, samRequestContext)
       groupEmails <- directoryDAO.loadGroups(groups, samRequestContext)
-      policyEmails <- policyMembers.toList.parTraverse((resourceAndPolicyName: FullyQualifiedPolicyId) => accessPolicyDAO.loadPolicy(resourceAndPolicyName, samRequestContext))
+      policiesAndEmails <- policyMembers.toList.parTraverse { (resourceAndPolicyName: FullyQualifiedPolicyId) =>
+        accessPolicyDAO.loadPolicy(resourceAndPolicyName, samRequestContext).map { accessPolicyOption =>
```

Comment:
is there a more efficient way to do this, instead of calling `accessPolicyDAO.loadPolicy` N times? I didn't see a "batch load policies" function.